src/dscanner/analysis/run.d

```diff
@@ -511,6 +512,10 @@ MessageSet analyze(string fileName, const Module m, const StaticAnalysisConfig a
 		checks ~= new IfConstraintsIndentCheck(fileName, tokens,
 		analysisConfig.if_constraints_indent == Check.skipTests && !ut);
 
+	if (moduleName.shouldRun!"trust_too_much"(analysisConfig))
+		checks ~= new TrustTooMuchCheck(fileName,
+		analysisConfig.trust_too_much == Check.skipTests && !ut);
```

Comment:
BTW interested in reducing this boiler-plate with a `static foreach`?
I know that some classes use a different constructor, but we be trivial for most, e.g.

```
enum checks = [
   tuple("trust_too_much", TrustTooMuchCheck),
];
static foreach (check; checks)
...
```